Coffee Time

Fiery36 opened the pull request:
Re-added the pyrolyse oven recipes for roasted coffee beans.

bruberu reviewed (commented):
I feel like these changes could probably be put into an already existing file, specifically GregTechFoodOption.groovy.

loxoDev commented:
Yeah, don't just put it in a random file.

Fiery36 committed:
Coffee Time

Re-added the pyrolyse oven recipes for roasted coffee bean, making coffee now obtainable

Fiery36 commented:
I've gone ahead and moved it to GregTechFoodOption.groovy

loxoDev commented:
👍 Good work
